buildspec-windows.yml

```diff
@@ -3,6 +3,13 @@ version: 0.2
 phases:
   install:
     commands:
+      - |
+        # Codebuild sets up a directory whose name changes with every build.
+        # We create a symlink with a fixed name to enable clcache to get cache hits.
+        Set-Location -Path "c:\codebuild\tmp"
+        New-Item -ItemType Junction -Name builddir -Value $env:CODEBUILD_SRC_DIR
+        Set-Location -Path "c:\codebuild\tmp\builddir"
```

Comment:
This doesn't seem to be necessary: use `$env:CLCACHE_BASEDIR = (Get-Item -Path ".\").FullName` instead.